### PR TITLE
Feat/improve docker

### PR DESCRIPTION
**Pull request by 20uf**
Hey,

- I optimized docker, I added the notion of context to be able to easily switch to version (7.2, ...)
- I modified the image on a new database which is Ubuntu
- I added a makefile in order to easily manipulate the container
- I updated the documentation

Regards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +22,98 @@ $ php certificationy.php
...
+Docker version 1.12.4, build 1564f02
+```
+> You must use the minimum version 1.12 of Docker.
+
+To install the docker-composer, please also refer to the official documentation: https://docs.docker.com/compose/install/.
+
+Once docker-composes installed (install it globally to be able and access from anywhere), to check its proper functioning, run docker-compose -v, you should get something like suit:
+
+```
+$ Docker-compos -v
```
Comment (mickaelandrieu) [inline README.md]:
```diff
- Docker-compos -v
+ docker-compose -v
```

File: README.md [hunk 2]
```diff
@@ -20,13 +22,98 @@ $ php certificationy.php
...
+- [Docker][docker] (1.12+)
+- [Docker-composer][docker-compose] (1.10+)
+- [GNU make][make]
+
+To install Docker, refer to the official documentation for your operating system: https://docs.docker.com/engine/installation/.
+
+Once Docker installed, to check its smooth running, run docker -v, you should get something like suit:
+
+```
+$ Docker -v
```
Comment (mickaelandrieu) [inline README.md]:
```diff
- Docker
+ docker
```

File: docker/7.1/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:16.04
+
+ENV LANG="en_US.UTF-8" \
+    LC_ALL="en_US.UTF-8" \
+    LANGUAGE="en_US.UTF-8" \
+    TERM="xterm" \
+    DEBIAN_FRONTEND="noninteractive" \
+    SYMFONY_ALLOW_APPDEV=1 \
+    NODE_VERSION=6.11.0 \
```
Comment (mickaelandrieu) [inline docker/7.1/Dockerfile]: you probably don't need `SYMFONY_ALLOW_APPDEV=1` and `NODE_VERSION=6.11.0` right?

Comment (20uf) [inline docker/7.1/Dockerfile]: Fixed

File: README.md (hunk 2 above)
Comment (20uf) [inline README.md]: Fixed

File: README.md (hunk 1 above)
Comment (20uf) [inline README.md]: Fixed

**Comment by mickaelandrieu** [comment]
Hi @20uf! 
so, what is your opinion about use of alpine instead of ubuntu 16.04? Should I merge this "as it" or are you still working on alpine docker file? :)

Mickaël

**Comment by 20uf** [comment]
Hey @mickaelandrieu ,

I'll take care of it tomorrow.

Mike

**Comment by 20uf** [comment]
I open another PR, this one is not valid.

**Pull request closed without merging**